### PR TITLE
Update README.md - T3AS

### PR DESCRIPTION
**Pull request by ireun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ files in `bin/`:
 - `t3a_dump_sram_init_boot_v1.26.bin`: SRAM content written by armlib/Keil init code (bootloader)
 - `t3a_dump_sram_init_main_v1.26.bin`: SRAM content written by armlib/Keil init code (main firmware)
 
-T3AS firmware is for GD32F305/GD32F307 version of the board, which wasn't released (yet?).
+T3AS firmware is for GD32F305/GD32F307 version of the board, which [was](https://www.aixuntech.com/newsinfo/aixun-new-product-launch-t3as-allinone-200w-soldering-station/) released in February of 2023.
```
Comment (c0d3z3r0) [inline README.md]: `which was [released](https://www.aixuntech.com/newsinfo/aixun-new-product-launch-t3as-allinone-200w-soldering-station/)`

**Review by c0d3z3r0 — CHANGES_REQUESTED** [review]

**Review by c0d3z3r0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (c0d3z3r0) [reply README.md]: oops, "in February of 2023" got missing

Comment (c0d3z3r0) [reply README.md]: fixed

**Comment by ireun** [comment]
@c0d3z3r0 do you use Discord? Or is there any other way of contacting You?

**Comment by c0d3z3r0** [comment]
No Discord, but feel free to drop me a pm on eevblog forums or libera.chat IRC

**Pull request merged**